docs/_sass/_main.scss

```diff
@@ -485,16 +486,12 @@ button {
     }
 }
 
-.icon {
-    color: $color-gray-green;
-    font-size: larger;
-    display: inline;
-
-    &.fa-angle-right {
-        padding-left: 4px;
-    }
+.base-icon {
+    width: 20px;
+    height: 20px;
 }
 
+
```

Comment:
Unnecessary Line break

Reply:
@Santhosh-Sellavel I just fixed it. Please help to check again